src/jira/client/jira-client.ts

```diff
@@ -388,6 +388,15 @@ export const getJiraClient = async (
 				logger.info("Sending remoteLinks payload to jira.");
 				await instance.post("/rest/remotelinks/1.0/bulk", payload);
 			}
+		},
+		linkedWorkspace: {
```

Comment:
Should we rather add it to model/jira-client (aka "the new Jira client")?

Reply:
We should also probably align this with what @Blastoplex is doing in his PR here https://github.com/atlassian/github-for-jira/pull/2232/files#diff-faa58cca2aee07a99f832ce04e13f7a1789ee66089c468de66644980486a600fR432

Reply:
@mjordan-atlassian, we need to use new jira-client to add new endpoint.  @Blastoplex also need to move his stuff in `model/jira-client` file.

Reply:
Oh cool, i see! didnt realise there was a good or bad client :p